config_examples/spanmetrics.yaml

```diff
@@ -0,0 +1,71 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+
+processors:
+  transform:
+    metric_statements:
+    - context: metric
+      statements:
+      # Extract sum and count metrics from the service. We can use these metrics
+      # to get the average request duration by using a metric query like so:
+      # requests.duration_sum:splitBy("service.name")/requests.duration_count:splitBy("service.name")
+      # Note that the resulting values in the above query will be unitless.
+      - extract_count_metric(false) where type == METRIC_DATA_TYPE_HISTOGRAM
+      - extract_sum_metric(false) where type == METRIC_DATA_TYPE_HISTOGRAM
+  filter:
+    metrics:
+      metric:
+      # The Dynatrace OTLP metrics ingest doesn't currently support histograms, so filter them out.
+      - type == METRIC_DATA_TYPE_HISTOGRAM
+  transform/spanmetrics:
+    metric_statements:
+    - context: datapoint
+      statements:
+      # Not required, but simplifies queries in Dynatrace by removing the requirement for a `.count`
+      # suffix on the metric name in the query.
+      - convert_sum_to_gauge() where IsMatch(metric.name, "^requests.*_sum") and metric.type == METRIC_DATA_TYPE_SUM
```

Comment:
I think I'd still leave this out. Adding a `.count` suffix isn't the end of the world and this confuses the example in my opinion. If sums are difficult to work with that's a product question, not something we should be papering over by ingesting semantically incorrect data.

Reply:
We should also consider a change to our [histogram conversion snippet](https://github.com/Dynatrace/dynatrace-otel-collector/blob/main/config_examples/split_histogram.yaml#L19) if we want to alter our overall approach here, I'm largely trying to build on and align with that.

I do see your point, I'll remove this for now and we can always add it back in later if we want. My counterpoint to making the metric type semantically incorrect is that this is a workaround for the lack of support for ingesting histograms, so we can bend the rules a little bit. Even if the sum is not an OTLP gauge, it may be semantically close enough to a Dynatrace gauge that we can make that leap in logic.